Resources/config/routing.yml

```diff
@@ -14,3 +14,8 @@ template_yui_module:
 ez_pjax:
     resource: "@eZPlatformUIBundle/Resources/config/routing_pjax.yml"
     prefix:   /pjax
+
+yui_combo_loader:
+    path: /combo
```

Comment:
That is true. I've used route name suggested in Jira issue.

Any suggestions? /cc @dpobel @andrerom